config/default.yml

```diff
@@ -77,17 +89,8 @@ Betterment/UnsafeJob:
 Betterment/UnscopedFind:
   StyleGuide: '#bettermentunscopedfind'
 
-Betterment/RedirectStatus:
-  SafeAutoCorrect: false
-  Description: Detect missing status codes when redirecting POST, PUT, PATCH, or DELETE responses
-  Include:
-    - app/controllers/**/*.rb
-
-Betterment/RenderStatus:
-  SafeAutoCorrect: false
-  Description: Detect missing status codes when rendering POST, PUT, PATCH, or DELETE responses
-  Include:
-    - app/controllers/**/*.rb
+FactoryBot/AssociationStyle:
```

Comment:
__comment:__ The `rubocop-factory_bot` gem is not a dependency of betterlint, so in a project that does not use that gem this entry causes an error like the following to be thrown by rubocop:

```
Error: unrecognized cop or department FactoryBot/AssociationStyle found
```